enhancements/sig-architecture/35-work-placement/work-placement.md

```diff
@@ -0,0 +1,192 @@
+
+### Non-Goals
+
+* Initially out of scope:
+  * Replica distribution
+    * A user can choose how to distribute a resource. For example, the user can define a deployment and declare to replicate the deployment to every cluster, or to distribute the replicas of the deployment to each cluster.
+    * A user can choose which cluster to deploy the workload.
+
+  * Fallback and reschedule
+    * When a workload cannot be fully run on a spoke (lack of resources, image pull backoff). It should be able to reschedule to another cluster.
+    * When a cluster was added/removed from a PlacementDecision, it should be able to reschedule.
+      * Including the changes upon the Placement’s matching policy.
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+User wants to distribute work (kubernetes resources) to a number of clusters in their fleet, filtering this group of clusters with placement.
+
+#### Story 2
+User removes a distributed work (distributed to clusters in the placement decision), the resources should be cleaned up similar to exisitng work behaviours
+
+#### Story 3
+Work placement should contain enough status information to determine if the manifest(s) were successfully applied to the clusters in the placement decision.
+
+### Implementation Details/Notes/Constraints
+
+* The user needs to create the resource in the workspace ns on the hub cluster. 
+* The controller will pick the cluster based on the placement decision.
+* The controller will read the resources and generate ManifestWork on the related cluster.
+* `[NON-GOLA]` If a user specifies a placement ref, the controller will use this placement to find clusters. Otherwise the controller will create a new empty placement that selects all clusters scoped in this ns. The user can mutate the placement spec afterwards for advanced cluster selection.
+* `[NON-GOLA]` By default, the controller will replicate the manifest to each manifestwork, if the mode is split. The controller will split the replica field and generate manifests with different replicas in different clusters (similar as splitter in kcp).
+* `[NON-GOLA]` The controller will check the status of the applied resources in the manifestwork, if the resource is degraded (e.g. deployment available replica is not enough). The controller will exclude this cluster and try to select another one.
+
+#### API
+```yaml
+apiversion: work.open-cluster-management.io/v1alpha1
+kind: PlacementWork # ApplicationWork?
+metadata:
+  name: app
+  namespace: default
+spec:
+  manifestWorkSpec:
+    workloads:
+      manifest:
+      - apiVersion: v1
+        kind: Service
+        metadata:
+          name: test01
+          namespace: default
+        spec: … 
+      - apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: test02
+          namespace: default
+        spec: … 
+  ## The following is a spike ##
+  distributionStrategy:
```

Comment:
can we elaborate more on the distributionStrategy ? what is the type Split ? and why is the distributionStrategy is repeated under test02 item as the Type is split ?

Reply:
Once we have the base controller, we can start working on the strategy spike.